Task3/EEL_task_3/src/main/java/com/epam/esm/giftcertificate/controller/GiftCertificateHateoasController.java

```diff
@@ -20,15 +24,27 @@ public class GiftCertificateHateoasController {
 
     private final GiftCertificateService giftCertificateService;
 
+
     public GiftCertificateHateoasController(GiftCertificateService giftCertificateService) {
         this.giftCertificateService = giftCertificateService;
     }
 
+    @GetMapping
+    public CollectionModel<GiftCertificateHateoas> showAll(@RequestParam(required = false, defaultValue = DEFAULT_PAGE) Integer page, @RequestParam(required = false, defaultValue = DEFAULT_SIZE) Integer size) {
```

Comment:
Please split parameters  by new line when string is more than 120-140 characters